core/backend/src/BlobContainerService.ts

```diff
@@ -153,9 +174,12 @@ export namespace BlobContainer {
     /** query the Scope for a container */
     queryScope(container: AccessContainerProps): Promise<Scope>;
 
-    /** query the Metadata for a container */
+    /** query the Metadata for a specific container */
     queryMetadata(container: AccessContainerProps): Promise<Metadata>;
 
+    /** Returns all containers and their metadata associated with a given iTwinId. Can be further queried by label and containerType. */
+    queryContainersMetadata?(userToken: UserToken, args: QueryContainerProps): Promise<MetadataResponse[]>;
```

Comment:
It is required because there is no other method on `BlobContainer.ContainerService` which does return `containerId`, except `create`. `Create` is a onetime call then afterward to retrieve `cotainerId` subsequently, we need some function. This is where queryContainersMetadata is required.

Reply:
It is required because there is no other method on `BlobContainer.ContainerService` which does return `containerId`, except `create`. `Create` is a onetime call then afterward to retrieve `containerId` subsequently, we need some function. This is where `queryContainersMetadata` is required.

Reply:
swapped to required